repository/src/main/resources/db/changelog/db.changelog-rdm-1240.3.xml

```diff
@@ -0,0 +1,83 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="rdm-1240-3" author="giri.benadikar@hmcts.net">
+        <insert tableName="case_field">
+            <column name="reference" value="jurisdiction"/>
+            <column name="live_from" value="${now}"/>
+            <column name="label" value="Jurisdiction"/>
+            <column name="hidden" value="false"/>
+            <column name="security_classification" value="PUBLIC"/>
+            <column name="field_type_id"
+                    valueComputed="(select id from field_type where reference = 'Text' and version = 1 and jurisdiction_id is null)"/>
```

Comment:
It may not be relevant at the moment as we only have 1 version of base types, but a better approach would be to remove `and version = 1` and replace it by:
```
order by version desc limit 1
```
to make sure we are using the latest version of the global field type.

Reply:
Done